<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BLAS Switcher</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1">
   <item>
    <widget class="QLabel" name="currentBlasText">
     <property name="text">
      <string>Current BLAS: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="refBlasOption">
     <property name="text">
      <string>Reference BLAS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="openBlasOption">
     <property name="text">
      <string>Open BLAS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
